ingest/importer/importer.py

```diff
@@ -70,7 +70,7 @@ def import_file(self, file_path, submission_url, is_update=False, project_uuid=N
             entity_map = EntityMap.load(spreadsheet_json)
             self.ingest_api.delete_submission_errors(submission_url)
             project = entity_map.get_project()
-            project and self.submitter.link_submission_to_project(project, submission_url)
+            project_uuid and project and self.submitter.link_submission_to_project(project, submission_url)
```

Comment:
yup, I think it's called shortcircuiting, it looks more concise than having if condition block.

Reply:
Indeed this is called short circuiting. Its purpose, however, is not to save writing an if statement, but to prevent evaluation of statement that would cuase errors. For example instead of:

```
print(1/n)
```
that would have problem when n is 0, you would have:
```
n and print(1/n)
```
so that the 2nd expression is evaluated only if the 1st one is true.

A by-product of this would be conditional execution, but this is unreadable and obscuring the intent of the writer from future reader.

Reply:
I see what you mean, let me see how I can improve this...

Reply:
these kind of syntax is so dependent on the language semantics and evaluation order, that's why i always get confused when switching from language to language. the advantage of if..else is that it is pretty standard/the same in most languages. I wonder if that makes up for the lack of "real" ternary operator in python...